Cargo.toml

```diff
@@ -25,25 +25,21 @@ lazy_static = "1.2"
 libc = "0.2"
 bitflags = "1.0"
 futures-preview = { version = "0.2", optional = true }
-glib-sys = { git = "https://github.com/gtk-rs/sys" }
-gobject-sys = { git = "https://github.com/gtk-rs/sys" }
+glib-sys = { git = "https://github.com/gtk-rs/sys", features = ["v2_40"] }
+gobject-sys = { git = "https://github.com/gtk-rs/sys", features = ["v2_42"] }
```

Comment:
This IMHO unneeded in final version

Reply:
Indeed, that was just for local testing :)